Sources/Error Handling/StoreKitError+Extensions.swift

```diff
@@ -36,6 +36,12 @@ extension StoreKitError: PurchasesErrorConvertible {
             return ErrorUtils.storeProblemError(error: self)
 #endif
 
+        #if compiler(>=6.1)
+        // StoreKitError.unsupported was introduced in iOS 18.4, which shipped with Xcode 16.3 beta 1 / Swift 6.1
+        case .unsupported:
+            return ErrorUtils.purchaseInvalidError(error: self)
```

Comment:
I'm open to using a different mapping function to map this to a `PurchasesError` - let me know if y'all think there's a more appropriate mapping available 👍